Error rendering item with recharge 'None'

**Describe the bug**  
Creating a custom item with limited uses and the Recharge type 'None' causes issues once added to a character sheet. The character sheet becomes inaccessible until the item is removed via console commands. Setting the Recharge type to 'Per Scene' or leaving it empty does not cause any issues.

Adding the item causes the sheet to immediately close and opening a character sheet with an affected item fails with an error that is only shown in the browser console:

`
foundry.mjs:27248 Uncaught (in promise) Error: Failed to render Application "CharacterSheet-Actor-vaihf0JdqLcpjr77":
Cannot read properties of undefined (reading 'label')
[No packages detected]
    at #render (foundry.mjs:27248:13)
    at async #try (foundry.mjs:7169:17)
Caused by: TypeError: Cannot read properties of undefined (reading 'label')
    at Object.<anonymous> (index.js:13557:83)
    at Object.c (handlebars.min.js:27:27865)
    at eval (eval at createFunctionContext (handlebars.min.js:29:10011), <anonymous>:10:234)
    at h (handlebars.min.js:27:25975)
    at CosmereItem.<anonymous> (handlebars.min.js:27:19411)
    at CosmereItem.c (handlebars.min.js:27:27865)
    at eval (eval at createFunctionContext (handlebars.min.js:29:10011), <anonymous>:10:52)
    at h (handlebars.min.js:27:25975)
    at c (handlebars.min.js:27:6943)
    at Object.<anonymous> (handlebars.min.js:27:7333)
    at Object.c (handlebars.min.js:27:27865)
    at eval (eval at createFunctionContext (handlebars.min.js:29:10011), <anonymous>:21:49)
    at h (handlebars.min.js:27:25975)
    at c (handlebars.min.js:27:6943)
    at Object.<anonymous> (handlebars.min.js:27:7333)
    at Object.c (handlebars.min.js:27:27865)
    at Object.eval [as main] (eval at createFunctionContext (handlebars.min.js:29:10011), <anonymous>:10:52)
    at c (handlebars.min.js:27:23534)
    at d (handlebars.min.js:27:23849)
    at e (handlebars.min.js:28:30886)
    at renderTemplate (foundry.mjs:65904:10)
    at async Object.renderComponent (index.js:2907:21)
    at async CharacterSheet._renderHTML (index.js:3326:34)
    at async #render (foundry.mjs:27239:22)
    at async #try (foundry.mjs:7169:17)
`

**To Reproduce**  
1. Create a custom item (I used the 'Equipment' type)
2. Go to 'Details'
3. Set 'Activation Type' = 'Utility'
4. Set 'Limited Uses' = 'Uses'
5. Set 'Recharge' = 'None'
6. Close the item editor
7. Add the item to a character sheet

**Expected behavior**  
The item should appear in the charactes equipment list.

**Screenshots**  

<img width="1102" height="573" alt="Image" src="https://github.com/user-attachments/assets/23568738-1617-433c-8562-39151df10a4e" />

**Foundry VTT Version**  
Foundry Version 13 Stable (Build version 351)
Cosmere Roleplaying Game Version 2.0.3

**Cosmere RPG System & Module Versions**  
|**Name**|**Version**|
| Cosmere Roleplaying Game | 2.0.3 |
| Cosmere RPG - Stormlight Handbook | 2.0.0 |
| Cosmere RPG - Stonewalkers | 2.0.0 |
| Cosmere RPG - Stormlight Worldguide | 2.0.0 |

**Browser**  
Observed in the embedded Foundry VTT browser (shoulde be Chrome, I think)

**Checklist:**
- [x] I have searched the existing issues to ensure this bug has not already been reported

--- a/src/system/utils/handlebars/index.ts
+++ b/src/system/utils/handlebars/index.ts
@@ -397,7 +397,7 @@ Handlebars.registerHelper(
                     const max = item.system.activation.uses.max;
                     const recharge = item.system.activation.uses.recharge;
 
-                    const hasRecharge = recharge != null;
+                    const hasRecharge = recharge != null && recharge !== 'none';
 
                     // Get config
                     const config =

